frameworks/agno/travel-planner/agentuity_agents/travel_planner/query_parser.py

```
@@ -0,0 +1,91 @@
+            model="gpt-4o",
+            messages=messages,
+            temperature=0
+        )
+
+        text = response.choices[0].message.content.strip()
+        try:
+            parsed = json.loads(text)
+        except json.JSONDecodeError:
+            match = re.search(r"\{.*?\}", text, re.DOTALL)
+            if not match:
+                raise ValueError("No JSON found in response") from None
```

Ruff (0.13.1)

Avoid specifying long messages outside the exception class

(TRY003)